packages/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -89,9 +89,10 @@ async function diffFolders (folderA: string, folderB: string) {
   if (stderr.length > 0)
     throw new Error(`Unable to diff directories. Make sure you have a recent version of 'git' available in PATH.\nThe following error was reported by 'git':\n${stderr}`)
 
-  const normalizePath = folderAN.startsWith('/')
-    ? (p: string) => p.slice(1)
-    : (p: string) => p
+  const normalizePath = (p: string) =>
+    (p.startsWith('/') || p.endsWith('/'))
+      ? p.replace(/^\/|\/$/g, '')
```

Comment:
slashes in the middle should also be removed?

Reply:
Just at the beginning / end, so that the `stdout` regex is able to replace the path correctly. If we replaced all slashes, the regex wouldn't work